docs/reference/glossary/network-latency-management.md

```diff
@@ -0,0 +1,112 @@
+---
+id: network-latency-management
+title: Network Latency Management
+sidebar_label: Network Latency Management
+---
+
+Every player is affected by a certain amount of latency, which is the result of the distance to the server/host, the number of hops a packet has to make, and the server's tick rate and update rate.
+
+This is where Network Latency Management comes in. Due to these delays, you would see where a player was a few milliseconds ago, not where they are right now. If the game didn't compensate for at least a portion of this delay, then you would be unable to hit other players.
+
+
+
+## Server rewind
+
+The server uses the latency of the player (including any inherent delay due to interpolation) to rewind time by an appropriate amount in order to determine what the shooting client saw at the time the shot was fired.
+ 
+
+## Interpolation
+
+A ------> B
+
+Interpolation works by essentially buffering a game state and rendering the game state to the player with a small, constant delay. When a packet from the server arrives, before immediately updating the position of an object , the client will interpolate the position, starting from the last known position. 
+
+Over an interpolation interval, the object will be rendered  smoothly  moving between the two positions. 
+
+## Extrapolation
+
+
+A ---- B -----> 
+
+Extrapolation is an attempt to estimate a future game state. On receipt of a packet from the server, the position of an object is updated to the new position. Awaiting the next update, the next position is extrapolated based on the current position and the movement at the time of the update. 
+
+The client will normally assume that a moving object will continue in the same direction. When a new packet is received, the position may be updated.
+
+## Action casting
```

Comment:
I have never heard of this term before in netcode? Where did we get that from?

Reply:
it was in a doc that i was given about netcode terms

Reply:
I have removed the section and reorganised the text

Reply:
title removed and doc restructured